modules/metering-uninstall-crds.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-uninstall.adoc
+
+[id="metering-uninstall-crds_{context}"]
+= Uninstalling metering CustomResourceDefinitions
+
+The metering CustomResourceDefinitions (CRDs) remain in the cluster after the Metering
+Operator is uninstalled and the `openshift-metering` namespace is deleted.
+
+[IMPORTANT]
+====
+Deleting the metering CRDs disrupts any additional metering installations in other namespaces.
```

Comment:
Again questioning the module with the wording about uninstalling metering from your namespace. Because this whole assembly seems to be removing it from the entire cluster.